er_aws_kms/__main__.py

```diff
@@ -0,0 +1,58 @@
+import os
+from pathlib import Path
+
+from external_resources_io.input import parse_model, read_input_from_file
+
+from er_aws_kms.input import AppInterfaceInput
+
+
+def get_ai_input() -> AppInterfaceInput:
+    """Get the AppInterfaceInput from the input file."""
+    return parse_model(
+        AppInterfaceInput,
+        read_input_from_file(
+            file_path=os.environ.get("ER_INPUT_FILE", "/inputs/input.json"),
+        ),
+    )
+
+
+def create_tf_vars_json(
+    app_interface_input: AppInterfaceInput, vars_file: str = "./module/tfvars.json"
```

Comment:
`vars_file` should be configurable via environment variable

Reply:
why?

Reply:
1. I don't like hard-coded things
2. We may want or need to adapt that for debugging purposes or testing.

Reply:
I added it, but I think this is completely unnecessary. 
* files need to be inside the module directory 
* You can override the value for testing purposes.